src/main/java/banking/dao/AccountDAO.java

```diff
@@ -0,0 +1,150 @@
+package banking.dao;
+
+import banking.configuration.DBManager;
+import banking.model.Account;
+import banking.model.Card;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class AccountDAO {
+
+    private final static String INSERT_INTO_ACCOUNT = "INSERT INTO account (number, pin) VALUES (?, ?)";
+    private final static String SELECT_CARD_NUMBER = "SELECT count(number) FROM account WHERE number = (?)";
+    private final static String SELECT_ALL = "SELECT * FROM account";
+    private final static String GET_VALID_CARD = "SELECT * FROM account WHERE number=? AND pin=?";
```

Comment:
Update constant name, it returns account now.

Reply:
fixed